src/main/java/io/singularitynet/Client/ClientStateMachine.java

```diff
@@ -640,6 +640,7 @@ public static void exitJava() {
             public void run() {
                 MinecraftClient.getInstance().getServer().stop(true);
                 MinecraftClient.getInstance().stop();
+                MinecraftClient.getInstance().stop();
```

Comment:
Why you call stop second time?

Reply:
@noskill  that is weird. I'll delete it

Reply:
Because it doesn't stops for me when not called. I've met this bug on my side multiple times. The only solution that worked was to call this methon twice. As I know, Innokenty couldn't reproduce this bug. Can you @noskill try this too? I want to test this on some other Linux machine too.

Reply:
When it fails? I think there is no tests for this code.